project/Dependencies.scala

```diff
@@ -21,12 +21,11 @@ object Dependencies {
     val declineCore = decline()
     val declineEffect = decline("effect")
     val declineRefined = decline("refined")
-    val requests = "com.lihaoyi" %% "requests" % "0.8.0"
-    val upickle = "com.lihaoyi" %% "upickle" % "3.1.3"
+    val requests = "com.lihaoyi" %% "requests" % "0.9.0"
+    val upickle = "com.lihaoyi" %% "upickle" % "3.3.1"
```

Comment:
I couldn't see where upickle is used but we alrady have circe for JSON handling in the project, so do we need to add another dependency for this? 

We also have http4s available from tessellation, so could requests be removed in lieu of using the HttpClient like [this](https://github.com/Constellation-Labs/flight-data/blob/main/source/project/flight-event-tracker/modules/data_l1/src/main/scala/io/constellationnetwork/data_l1/daemon/OnchainLatestLookup.scala)?

Reply:
Fixed